README.org

```diff
@@ -48,6 +48,16 @@ Lispyville can also be used without lispy:
 ;; ...
 #+end_src
 
+If you're using `use-package` and `general`. The hooks and key-themes in the
```

Comment:
I'd just say "Here is an example configuration with use-package and general that adds additional keybindings." Also, use = instead of ` (since org mode).